package.json

```diff
@@ -62,6 +62,7 @@
 		"onCommand:cosmosDB.deleteDocDBDocument",
 		"onCommand:cosmosDB.deleteGraphDatabase",
 		"onCommand:cosmosDB.deleteGraph",
```

Comment:
![image](https://user-images.githubusercontent.com/6913354/46633049-6a158080-cb01-11e8-97d1-3143c037f872.png)
Command is somewhat vague when out of context.  How about "Import into MongoDB/CosmosDB Collection"?

Reply:
I guess that's then confusing when on the collection node:
![image](https://user-images.githubusercontent.com/6913354/46633233-ec05a980-cb01-11e8-8bf1-d095add87394.png)
Maybe we need two commands so they can have different wording?

Reply:
Filing an issue for this.